### PR TITLE
refactor wallets ui

### PR DESCRIPTION
**Pull request by buck54321**
Normalize styling. Make responsive.

Purge most sans-light. Will do entire front end in a followup.

https://github.com/decred/dcrdex/assets/6109680/56441495-73ee-4bee-bf0e-529b9279d2f3

### CODE REVIEW

**Review by martonp — COMMENTED** [review]
If there are initially no transactions and I make a transaction, the transaction history table doesn't show up. Otherwise, looks great.

**Review by JoeGruffins — APPROVED** [review]
Looks really great.

**Comment by peterzen** [comment]
The uniform border colors (layout columns - tables - buttons) don't suggest visual hierarchy, and there are many many lines the eye must navigate.  With slightly different border color brightness applied the eyeballs would have an easier time.

image

File: client/webserver/site/src/html/wallets.tmpl [hunk 1]
```diff
@@ -196,93 +205,91 @@
...
                     </tr>
                     <tr id="txHistoryDateRow">
                       <td colspan="4" class="text-center grey">
-                        <span class="fs18" data-tmpl="date"></span>
+                        <span class="fs15 demi" data-tmpl="date"></span>
                       </td>
                     </tr>
                   </tbody>
                 </table>
                 <span id="earlierTxs">Load earlier transactions...</span>
```
Comment (peterzen) [inline client/webserver/site/src/html/wallets.tmpl:282]: Small vertical rhythm issue - `pt-2` would fix

**Comment by buck54321** [comment]
> If there are initially no transactions and I make a transaction, the transaction history table doesn't show up. Otherwise, looks great.

I'm trying to reproduce but I'm not having luck. I'm creating a native btc wallet on a fresh simnet harness and sending it funds from alpha. Page is updating correctly.

**Comment by buck54321** [comment]
image

File: client/webserver/site/src/js/wallets.ts [hunk 2]
```diff
@@ -1561,43 +1561,57 @@ export default class WalletsPage extends BasePage {
...
+      lastSubLockedRow.classList.add('sub')
+    }
+    const setRowClasses = () => {
+      if (!lastSubLockedRow) return
+      (lastPrimaryRow as PageElement).classList.add('itemized')
+      lastSubLockedRow.classList.add('last')
     }
+
+    addPrimaryBalance(intl.prep(intl.ID_AVAILABLE_TITLE), bal.available, '')
+    if (bal.other.Shielded !== undefined) {
```
Comment (martonp) [inline client/webserver/site/src/js/wallets.ts:1596]: ETH does not have an `other` section, so this causes an error.

**Comment by martonp** [comment]
> I'm trying to reproduce but I'm not having luck. I'm creating a native btc wallet on a fresh simnet harness and sending it funds from alpha. Page is updating correctly.

Nvm, seems to be working now.

**Comment by buck54321** [comment]
Made some minor stylistic changes to normalize across other recent work. Without any change requests, I will plan to merge this in a couple of days.

**Pull request merged**